packages/utils/src/ethereumHelper.ts

```diff
@@ -9,10 +9,7 @@ export async function getSignature(
   const address = await signer.getAddress();
   if (!ethereum.request) throw new Error('No `request` On Ethereum Provider');
 
-  let params = [msg, address.toLowerCase()];
-  if (ethereum.isMetaMask) {
-    params = [params[1], params[0]];
-  }
+  let params = [msg, address];
```

Comment:
I apologize for missing this, but `let` should be `const` here since we no longer alter `params`. I'm surprised the linter didn't complain. (It did for me.)

```suggestion
  const params = [msg, address];
```

Reply:
done